packages/react/src/components/AccountSettings/ChangePassword/ChangePassword.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+
+import { Logger } from 'aws-amplify';
+import { changePassword, translate } from '@aws-amplify/ui';
+
+import { useAuth } from '../../../internal';
+import { View, Flex } from '../../../primitives';
+import {
+  DefaultCurrentPassword,
+  DefaultError,
+  DefaultNewPassword,
+  DefaultSubmitButton,
+} from './defaultComponents';
+import { ChangePasswordProps } from './types';
+import { FormValues } from '../types';
+
+const logger = new Logger('ChangePassword');
+
+const ChangePassword: React.ComponentType<ChangePasswordProps> = (props) => {
```

Comment:
Unless there is requirement to type `ChangePassword` with `React.ComponentType`, we don't generally need need to use when typing components:
```suggestion
function ChangePassword(props: ChangePasswordProps): JSX.Element | null {
```

Basically `React.ComponentType` is useful for declaring override component params on type interfaces due to how generic/flexible it is, but since we know exactly what `ChangePassword` is (a functional component), we can skip using it here

Reply:
Understood! Adjusting.

Reply:
Made more explicit in 97936eec8e4fce7511c6c3c5deee9aed98b9984b: 

https://github.com/aws-amplify/amplify-ui/blob/97936eec8e4fce7511c6c3c5deee9aed98b9984b/packages/react/src/components/AccountSettings/index.ts#L1

Reply:
Done: https://github.com/aws-amplify/amplify-ui/blob/97936eec8e4fce7511c6c3c5deee9aed98b9984b/packages/react/src/components/AccountSettings/ChangePassword/ChangePassword.tsx#L19-L22